src/alexapi/device_platforms/dummy.py

```diff
@@ -11,36 +14,28 @@ def __init__(self, config):
 		self.should_confirm_trigger = False
 
 	def setup(self):
-		if self._pconfig['verbose']:
```

Comment:
I also use the _verbose_ option in the unmerged `hyperion` platform. Shall we keep it in the specific platform config or move it to a more general place?